theano/scalar/tests/test_basic.py

```diff
@@ -146,6 +146,17 @@ def test_many_outputs(self):
         fn = gof.DualLinker().accept(g).make_function()
         assert fn(1.0, 2.0, 3.0) == [6.0, 7.0, 0.5]
 
+    def test_composite_printing(self):
+        x, y, z = floats('xyz')
+        e0 = x + y + z
+        e1 = x + y * z
+        e2 = x / y
+        C = Composite([x, y, z], [e0, e1, e2])
+        c = C.make_node(x, y, z)
+        g = FunctionGraph([x, y, z], c.outputs)
+        fn = gof.DualLinker().accept(g).make_function()
+        assert str(g) == '[*1 -> Composite{((i0 + i1) + i2), (i0 + (i1 * i2)), (i0 / i1)}(x, y, z), *1::1, *1::2]'
```

Comment:
Also, put parentheses around  the string and break it in the middle, like

```python
('This is too long to fit on one line, and so we '
 'will instead break it across multiple lines.')
```
Make sure the quotation marks line up as per PEP8.